### PR TITLE
Jc/standardize trajectories 2

### PR DESCRIPTION
**Pull request by JulienCalbert**
- suppress Control folder
- standardize trajectories

### CODE REVIEW

File: docs/src/examples/DC-DC converter.jl [hunk 1]
```diff
@@ -82,12 +81,12 @@ concrete_controller = MOI.get(optimizer, MOI.RawOptimizerAttribute("concrete_con
...
 x0 = SVector(1.2, 5.6)
-x_traj, u_traj =
-    CO.get_closed_loop_trajectory(concrete_system.f, concrete_controller, x0, nstep)
+control_trajectory =
+    ST.get_closed_loop_trajectory(concrete_system.f, concrete_controller, x0, nstep)
 
 fig = plot(; aspect_ratio = :equal);
 plot!(concrete_system.X);
-plot!(UT.DrawTrajectory(x_traj))
+plot!(control_trajectory)
```
Comment (adrienbanse) [inline docs/src/examples/DC-DC converter.jl:89]: That's much better like that

File: src/system/trajectory.jl [hunk 2]
```diff
@@ -51,7 +51,47 @@ struct HybridTrajectory{T, TT, XVT <: AbstractVector{T}, UVT <: AbstractVector{T
...
 
-# Trajectory closed loop
+struct Trajectory{T}
+    seq::Vector{T}
+end
+
+Base.length(traj::Trajectory) = length(traj.seq)
+get_elem(traj::Trajectory, n::Int) = traj.seq[n]
+
+@recipe function f(traj::Trajectory)
```
Comment (adrienbanse) [inline src/system/trajectory.jl:61]: I think that removing DrawTrajectory would be cleaner here, TODO?

**Review by adrienbanse — APPROVED** [review]
That's great thanks a lot @JulienCalbert 
I would just remove DrawTrajectory, because now it's a useless middle layer for plotting, but we can open an issue and do it later

File: docs/src/examples/DC-DC converter.jl (hunk 1 above)
Comment (JulienCalbert) [reply docs/src/examples/DC-DC converter.jl:89]: Thanks ! I will merge and open an issue

**Pull request merged**